### ISSUE TITLE
Swiss-system tournament pairing

### ISSUE THREAD

**Issue by sovaz1997** [cutechess/cutechess#267]
Do you plan to develop the Swiss-system tournament system in the cutechess-cli (and gui)?

**Comment by ilaripih** [comment]
Yes, we do. Not enough time to do it at the moment, but it's an essential feature.

**Comment by alwey** [comment]
@sovaz1997 : please consider reopening this issue. Even if there is no direct action, feature requests usually stay open as a reminder. This issue also has been given the tags enhancement and lib. If the issue is closed, it is more difficult to remember and to separate from fulfilled feature requests.

**Comment by kanchess** [comment]
I strongly support the feature request, we are even considering to use the Swiss format at TCEC for certain occasions. 
Thank you in advance for helping making this work :)

**Comment by Krzychumag** [comment]
I am also asking for this function Swiss system.

**Comment by Krzychumag** [comment]
Can you once again ask for the opportunity to play Swiss tournaments?

**Comment by kanchess** [comment]
yes please, by now it is sure we will use this in TCEC as well, in bonus for starters, and perhaps more ideas and regular formats will grow out of this, when available.
thank you in advance <3

**Comment by Krzychumag** [comment]
I apologise that I am asking. What with Swiss System in the program Cutechess.
